platform/solr/platform-solr-server-standalone/src/main/webapp/js/angular/controllers/collections.js

```diff
@@ -52,7 +52,6 @@ solrAdminApp.controller('CollectionsController',
                   }
               }
               if ($routeParams.collection && !$scope.collection) {
-                  alert("No collection called " + $routeParams.collection)
```

Comment:
❓ Is there another mechanism in place to eliminate the need to alert the user using this approach?

Reply:
Not sure. If you feel it is important that the user knows that the collection does not exist, then I could put this line back

Reply:
This file is copied from solr I believe so we shouldn't be changing it.

Reply:
That goes for everything in platform-solr-server-standalone